runtime/compiler/codegen/J9CodeGenerator.cpp

```diff
@@ -1530,7 +1541,93 @@ J9::CodeGenerator::lowerTreeIfNeeded(
 
    }
 
+void
+J9::CodeGenerator::rerunLivenessAnalysis()
```

Comment:
The similar code is duplicated in GRA, GlobalLiveVariablesForGC and here, can we extract it to a util and call that util instead?

Reply:
They're all different slightly. `TR_GlobalRegisterAllocator` is in `OMR` and checks compilation object visit count but `TR_GlobalLiveVariablesForGC` doesn't. GRA doesn't mark collected locals initialized or uninitialized if they're at the start of the block, which GlobalLiveVariablesForGC does.